ci/utility/resources.yml

```
@@ -0,0 +1,31 @@
+resources:
+  - name: dataworks-s3-object-tagger
+    type: git
+    source:
+      branch: master
+      repository: dwp/dataworks-s3-object-tagger
+      uri: https://github.com/dwp/dataworks-s3-object-tagger.git
+    check_every: 5m
```

GitHub.com resources should use webhooks instead of polling
```suggestion
    check_every: 720h
```